core/templates/dev/head/css/oppia.css

```diff
@@ -122,6 +122,10 @@ md-input-group.md-default-theme label
   word-wrap: break-word;
 }
 
+.oppia-learner-refresher-exploration-confirmation-body {
```

Comment:
Put these new classes in the corresponding modal directive -- easier to keep track of them that way. (At some point we should break oppia.css up and put stuff into the relevant directives, otherwise everything gets loaded every time.)

Reply:
Done